src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java

```diff
@@ -115,6 +115,8 @@ public class GitLabSCMNavigator extends SCMNavigator {
     private boolean wantSubGroupProjects;
     private transient GitLabOwner gitlabOwner; // TODO check if a better data structure can be used
 
+    private boolean markUnstableAsSuccess;
```

Comment:
Not important, simple note: according to other [repository examples](https://github.com/REC-SPb-ETU/gitlab-branch-source-plugin/blob/master/src/main/java/io/jenkins/plugins/gitlabbranchsource/WebhookListenerBuildConditionsTrait.java#L33) and best practice of java programming it's better not to use default simple type values but assign them explicitly

Reply:
Fixed inside GitLabSCMSourceContext